geometry_msgs/Point[] imagePoints
---
geometry_msgs/Point[] worldPoints